### DISCUSSION TITLE
Could someone tell me what the "Narrow" and "shortedges" in headland do?

### DISCUSSION THREAD

**Discussion by rento112** [Courseplay/courseplay#7005] [Q&A] [answered]
Could someone tell me what the "Narrow" and "shortedges" in headland do?

**Comment by Tensuko** [comment, marked as answer]
Why you don't try it out ? It doesn't hurt...
It is a pattern for center part.

**Reply by rento112** [reply]
I took a look at it, but didnt seem all that impoortant. So I was wondering what it's for by asking
